docs/dynamic-programming/longest_common_subsequence.md

```diff
@@ -0,0 +1,121 @@
+
+# Longest Common Subsequence (LCS)
```

Comment:
We need on top

```md
---
id: ___
title: ___
sidebar_label: ___
sidebar_position: ___
description: ___
tags: [___, __, ]
---
```

### Detailed Guidelines

- id: Unique identifier for the route.
- title: Title of the problem.
- sidebar_label: Problem number or label for the sidebar.
- tags: Relevant tags for the problem.
- description: A short SEO-friendly description. No HTML tags.

Reply:
I am not sure what to add in id, sidebar_label. Can i provide them on my own? Or maybe see somewhere